### ISSUE TITLE
The URLs are not accessable

### ISSUE THREAD

**Issue by BierOne** [airsplay/lxmert#108]
Hello, airsplay!

Thanks for your generous sharing of this outstanding work!

However, the URLs provided in this repository seem not accessible anymore. Could you please update them? 

Thank you in advance.

The below is an example of this connection error.

```
--2021-10-10 21:43:33--  http://nlp.cs.unc.edu/data/lxrt_noqa/Epoch19_LXRT.pth%0D
Resolving nlp.cs.unc.edu (nlp.cs.unc.edu)... 152.2.128.53
Connecting to nlp.cs.unc.edu (nlp.cs.unc.edu)|152.2.128.53|:80... connected.
HTTP request sent, awaiting response... 503 Service Unavailable
2021-10-10 21:44:33 ERROR 503: Service Unavailable.
```

**Comment by airsplay** [comment]
Hi, 

It seems that our server on UNC is periodically maintained thus it would sometimes be unable to download. 

However, the server was rebuilt yesterday (on Oct 19) thus the downloading service might be down for a few days. I am now contacting the admins to recover the functionality. 

Best,
Hao

**Comment by BierOne** [comment]
Thanks for your kind reply! I am always looking forward to your update.

**Comment by airsplay** [comment]
Hi, 

The downloading service is back and you can test it now. Please let me know if you further have issues in accessing them. I will be inactive on Github in the near future thus you can directly reach me for these link issues. 

Best,
Hao

**Comment by BierOne** [comment]
Hello, Mr. Tan!

I am so glad to hear this good news.

However, the UNC link seems still not working. The following is the error
messages:

*wget https://nlp.cs.unc.edu/data/model_LXRT.pth
<https://nlp.cs.unc.edu/data/model_LXRT.pth> -P snap/pretrained*
--2021-10-22 11:57:50--  https://nlp.cs.unc.edu/data/model_LXRT.pth
Resolving nlp.cs.unc.edu (nlp.cs.unc.edu)... 152.2.128.53
Connecting to nlp.cs.unc.edu (nlp.cs.unc.edu)|152.2.128.53|:443... failed:
Connection timed out.
Retrying.


--2021-10-22 12:00:02--  (try: 2)
https://nlp.cs.unc.edu/data/model_LXRT.pth
Connecting to nlp.cs.unc.edu (nlp.cs.unc.edu)|152.2.128.53|:443... failed:
Connection timed out.
Retrying.


--2021-10-22 12:02:14--  (try: 3)
https://nlp.cs.unc.edu/data/model_LXRT.pth
Connecting to nlp.cs.unc.edu (nlp.cs.unc.edu)|152.2.128.53|:443... failed:
Connection timed out.


I would be highly appreciated if you could repair it.

Thank you!

Regards,
Yibing



Hao Tan ***@***.***> 于2021年10月22日周五 上午5:41写道：

**Comment by airsplay** [comment]
Hi,

Could you try with
`wget --no-check-certificate https://nlp.cs.unc.edu/data/model_LXRT.pth`.

I will contact the server manager to solve it.

Best,
Hao

**Comment by BierOne** [comment]
Thanks for your reply.

However, It still does not work. Thank you.


Hao Tan ***@***.***> 于2021年10月22日周五 下午1:04写道：

**Comment by airsplay** [comment]
Hmmm, that's strange. It works on my end now. I will check with them to solve it.

**Comment by airsplay** [comment]
Hi,

We have densely tested the link on our end (from multiple sites) and it works well here. Could you try with the following things?

1. try visiting the website nlp.cs.unc.edu
2. try ping the website by "ping nlp.cs.unc.edu"
3. confirm that your ca-certificates package is up to date

Best,
Hao

**Comment by BierOne** [comment]
Hi, Mr. Hao!

I have tried following your instructions. And I also tested it by using the
different networks from China mainland and Hong Kong.

However, the link nlp.cs.unc.edu is still not reachable.

Much appreciated for your kind help.

Best,
Yibing




Hao Tan ***@***.***> 于2021年10月23日周六 上午4:15写道：

**Comment by airsplay** [comment]
If you can not visit `nlp.cs.unc.edu`, it is more like a DNS or web caching issue.

For testing DNS, could you try visit `http://152.2.128.53/`.

For testing web cache, could you try using the incognito mode of the web browser? In that way, it will resolve the potential caching issue.

**Comment by BierOne** [comment]
Dear Mr. Hao,

Thank you so much for your kind help. I have tried the incognito mode and
the ip address. However, it's still not working.

Actually, what I really need is the model pretrained without vqa datasets,
which seems only can be downloaded from this link:
*http://nlp.cs.unc.edu/data/lxrt_noqa/Epoch19_LXRT.pth
<http://nlp.cs.unc.edu/data/lxrt_noqa/Epoch19_LXRT.pth>*. I'd highly
appreciate it if you could upload this model to your* google drive*, which
is reachable for me.

By the way, I also have a question about the model saved in
*https://nlp.cs.unc.edu/data/github_pretrain/lxmert20/Epoch20_LXRT.pth
<https://nlp.cs.unc.edu/data/github_pretrain/lxmert20/Epoch20_LXRT.pth>*.
Is this model pre-trained with the VQA-2.0 val set?

Because I observed many repositories using this pre-trained model on the
VQA-CP datasets, where the test set contains many overlapped QA pairs with
the VQA-2.0 val set. If so, this should be an unwanted behavior, since
their high performance may be obtained by peeking at test sets.

Best regards,
Yibing

Hao Tan ***@***.***> 于2021年11月1日周一 上午3:47写道：

**Comment by BierOne** [comment]
Hi, Tan!

I finally downloaded the models! Thank you so much for the help.

Could you please tell me if this model (https://nlp.cs.unc.edu/data/github_pretrain/lxmert20/Epoch20_LXRT.pth ) is pre-triained on the VQA validation set?  Thank you!
